cowprotocol/competitor_analysis/cow_alternatives_analysis_4810844.sql

```diff
@@ -8,16 +8,27 @@
 --  {{chain_of_interest}} - which chain is not used by cohort of users we want to investigate
 --  {{show_competitors_on_target_chain_only}} - filters competitor's usage to only {{chain_of_interest}}
 
-with all_cow_users as (
+with all_transactions as (
+    select
+        tx_from,
+        tx_hash,
+        blockchain,
+        project,
+        amount_usd,
+        block_time,
+        product_type
+    from
+        "query_4836358(start_time='{{start_time}}', end_time='{{end_time}}')"
+),
+
+all_cow_users as (
     select
         tx_from as address,
         array_agg(distinct blockchain) as uses_on_chains
     from
-        dex_aggregator.trades
+        all_transactions
```

Comment:
nit: no need for this cte, you should be able to directly do

```suggestion
        "query_4836358(start_time='{{start_time}}', end_time='{{end_time}}')"
```

here